### PR TITLE
Add SLIZ token

### PR DESCRIPTION
**Pull request by IrishLatte19**
The token is active on coinpaprika

### Contribution type
Please check the type of contribution this pull request is for:

- [ ] New spell(s)
- [ ] Adding to existing spell lineage
- [ ] Bug fix

**Note:** You can safely discard any section below which doesn't apply based on selection above

---

### For new spell(s)
If you are building new spell(s), please provide the following information:

- **Spell name(s):** *schema.table_name*
- **Description:** [Detailed description of the new spell(s) and their purpose]
- **Who are the new spell(s) for?** [Internal team or general community]
- **How will the new spell(s) be used downstream?** [Description of downstream usage]
- **Implementation details:** [Information on how the spell(s) are implemented]
- **Test instructions:** [How to test the new spell(s)]
- **Related issue(s):** [Link to related issues, if any]

---

### For adding to existing spell lineage
If you are adding to an existing spell lineage, please provide the following information:

- **Description:** [Description of the changes made]

---

### For bug fixes
If you are fixing a bug, please provide the following information:

- **Description:** [Description of the bug fix]
- **Steps to reproduce:** [How to reproduce the bug]
- **Implementation details:** [Information on how the bug was fixed]
- **Test instructions:** [How to test the fix]
- **Related issue(s):** [Link to related issues, if any]

---

### Additional information
Please provide any additional information that might help us review your pull request:

- [Any additional information]

---

Thank you for your contribution!

### CODE REVIEW

**Comment by IrishLatte19** [comment]
image
@Hosuke The token is active. My changes were commented by you. Am i missing something ? Please guide

**Comment by Hosuke** [comment]
> image @Hosuke The token is active. My changes were commented by you. Am i missing something ? Please guide

Sorry, I will recover this token.

File: models/prices/arbitrum/prices_arbitrum_tokens.sql [hunk 1]
```diff
@@ -197,6 +197,6 @@ FROM
     ('gyd-gyro-dollar','arbitrum','GYD',0xca5d8f8a8d49439357d3cf46ca2e720702f132b8, 18),
     ('a51-a51-finance','arbitrum','A51',0xb3f13b0c61d65d67d7d6215d70c89533ee567a91, 18),
     ('oseth-stakewise-staked-eth','arbitrum','osETH',0xf7d4e7273e5015c96728a6b02f31c505ee184603, 18),
-    -- ('sliz-solidlizard-dex-token','arbitrum','SLIZ',0x463913D3a3D3D291667D53B8325c598Eb88D3B0e, 18), -- Inactive
+    ('sliz-solidlizard-dex-token','arbitrum','SLIZ',0x463913D3a3D3D291667D53B8325c598Eb88D3B0e, 18),
```
Comment (Hosuke) [inline models/prices/arbitrum/prices_arbitrum_tokens.sql:200]: I am not sure but it shows ` "is_active": false,` in my view:
https://api.coinpaprika.com/v1/coins/sliz-solidlizard-dex-token

```
{"id":"sliz-solidlizard-dex-token","name":"SolidLizard Dex Token","symbol":"SLIZ","rank":0,"is_new":false,"is_active":false,"type":"token","contract":"0x463913D3a3D3D291667D53B8325c598Eb88D3B0e","platform":"arb-arbitrum","contracts":[{"contract":"0x463913D3a3D3D291667D53B8325c598Eb88D3B0e","platform":"arb-arbitrum","type":"Other"}],"logo":"https://static.coinpaprika.com/coin/sliz-solidlizard-dex-token/logo.png","parent":{"id":"arb-arbitrum","name":"Arbitrum","symbol":"ARB"},"tags":[{"id":"arbitrum-ecosystem","name":"Arbitrum Ecosystem","coin_counter":624,"ico_counter":1},{"id":"decentralized-exchange-dex","name":"Decentralized Exchange (DEX)","coin_counter":148,"ico_counter":0}],"team":[],"description":"","message":null,"open_source":false,"started_at":null,"development_status":null,"hardware_wallet":false,"proof_type":null,"org_structure":null,"hash_algorithm":null,"links":{"explorer":["https://arbiscan.io/token/0x463913D3a3D3D291667D53B8325c598Eb88D3B0e"],"medium":["https://medium.com/@solidlizardfi"],"website":["https://solidlizard.finance"]},"links_extended":[{"url":"https://discord.com/invite/fSdcAQcGxw","type":"discord"},{"url":"https://arbiscan.io/token/0x463913D3a3D3D291667D53B8325c598Eb88D3B0e","type":"explorer"},{"url":"https://medium.com/@solidlizardfi","type":"medium"},{"url":"https://t.me/solidlizardfi","type":"telegram","stats":{"members":500}},{"url":"https://twitter.com/solidlizardfi","type":"twitter","stats":{"followers":7517}},{"url":"https://solidlizard.finance","type":"website"}],"whitepaper":{"link":"https://solidlizard.gitbook.io/solidlizard/solidlizard-protocol/welcome","thumbnail":"https://static.coinpaprika.com/storage/cdn/whitepapers/11037299.jpg"},"first_data_at":"2023-03-19T00:00:00Z","last_data_at":"2024-06-24T04:23:00Z"}
```

Maybe it is not very stable.

**Review by Hosuke — APPROVED** [review]

File: models/prices/arbitrum/prices_arbitrum_tokens.sql (hunk 1 above)
Comment (IrishLatte19) [reply models/prices/arbitrum/prices_arbitrum_tokens.sql:200]: Ohhh okay! What can be done in such cases?
Will it create any issue in future?
@Hosuke

Comment (Hosuke) [reply models/prices/arbitrum/prices_arbitrum_tokens.sql:200]: I think we can add this token, should be fine.

Comment (IrishLatte19) [reply models/prices/arbitrum/prices_arbitrum_tokens.sql:200]: Thanks a lot!!

**Review by jeff-dude — APPROVED** [review]

**Pull request merged**